docs/getting-started/running-a-validator.md

```diff
@@ -135,7 +135,7 @@ Add a seed node for initial peer discovery.
 *For Regen Mainnet:*
 
 ```bash
-PERSISTENT_PEERS="aebb8431609cb126a977592446f5de252d8b7fa1@regen.rpc.vitwit.com:26656"
+PERSISTENT_PEERS="c4460b52c34ad4f12168d05807e998bb8e8b4812@mainnet.regen.network:26656,aebb8431609cb126a977592446f5de252d8b7fa1@regen.rpc.vitwit.com:26656"
```

Comment:
who is this additional peer?

Reply:
This is an RND full node operated by the RND team.